backend/docker-services/setup-configuration/data.yaml

```diff
@@ -0,0 +1,40 @@
+zgw_consumers_config_enable: True
+zgw_consumers:
+  services:
+  - identifier: zaken-test
+    label: Open Zaak - Zaken API
+    api_root: http://localhost:8003/zaken/api/v1/
+    api_type: zrc
+    auth_type: zgw
+    client_id: test-vcr
+    secret: test-vcr
+  - identifier: documenten-test
+    label: Open Zaak - Documenten API
+    api_root: http://localhost:8003/documenten/api/v1/
+    api_type: drc
+    auth_type: zgw
+    client_id: test-vcr
+    secret: test-vcr
+  - identifier: catalogi-test
+    label: Open Zaak - Catalogi API
+    api_root: http://localhost:8003/catalogi/api/v1/
+    api_type: ztc
+    auth_type: zgw
+    client_id: test-vcr
+    secret: test-vcr
+  - identifier: besluiten-test
+    label: Open Zaak - Besluiten API
+    api_root: http://localhost:8003/besluiten/api/v1/
+    api_type: brc
+    auth_type: zgw
+    client_id: test-vcr
+    secret: test-vcr
```

Comment:
Yes! I was also thinking that it is handy for us to have this because it configures the services automatically to talk to the test Open Zaak that can be started with the docker-compose in the docker-services folder.

Reply:
This is also useful if we want to configure our local environment to talk to the services that are in the docker-services folder!